api/scpca_portal/templates/readme/metadata_only.md

```diff
@@ -0,0 +1,55 @@
+{% load custom_filters %}
+Generated on: {{ date }}
+
+# Alex's Lemonade Stand Foundation Single-cell Pediatric Cancer Atlas
+
+The [Single-cell Pediatric Cancer Atlas](https://scpca.alexslemonade.org) is a database of single-cell and single-nuclei data from pediatric cancer clinical samples and xenografts, built by the [Childhood Cancer Data Lab](https://www.ccdatalab.org/) at [Alex's Lemonade Stand Foundation](https://www.alexslemonade.org/).
+
+## Contents
+
+{% if entire_portal %}
+This download includes associated metadata for samples from all projects currently available at time of download in the ScPCA portal.
+{% else %}
+This download includes associated metadata for samples from project {{ included_projects|markdown_links }} in the ScPCA portal.
```

Comment:
```suggestion
{% for proejct_accession, project_url in included_projects %}
This download includes associated metadata for samples from project [{{ project_accession }}]({{ project_url }}) in the ScPCA portal.
{% endfor %}
```

Let's just keep this simple. For now we only anticipate two use cases:
- All projects
- One project

We can rewrite it to make sense for an arbitrary amount of projects later.